docs/app/documentation/component-docs/calendar/examples/calendar-single-example.component.ts

```diff
@@ -19,20 +19,20 @@ export class CalendarSingleExampleComponent {
 
     date = FdDate.getToday();
 
-    myDisableFunction = function(d: FdDate): boolean {
+    myDisableFunction = function (d: FdDate): boolean {
         const day = d.getDay();
         return day === 6 || day === 7;
     };
 
     // Block days before/after any day
-    myBlockFunction = function(d: FdDate): boolean {
-        const firstDay = new FdDate(2019, 7, 21);
-        const lastDay = new FdDate(2019, 7, 30);
+    myBlockFunction = function (d: FdDate): boolean {
+        const firstDay = FdDate.getToday();
+        const lastDay = new FdDate(firstDay.year, firstDay.month, firstDay.day + 7);
```

Comment:
Please use `FdDate.getToday().nextDay()` There will be problems when day is above 24 or 25.

Reply:
@JKMarkowski I have already looked into this situation, and it seems to behave just fine, proper dates are being crossed-out when blocked. See screenshots. 

<img width="849" alt="Screen Shot 2019-08-07 at 8 42 51 AM" src="https://user-images.githubusercontent.com/25252963/62623651-63a59000-b8ef-11e9-8416-5a7b6f71551f.png">
<img width="419" alt="Screen Shot 2019-08-07 at 8 42 44 AM" src="https://user-images.githubusercontent.com/25252963/62623655-656f5380-b8ef-11e9-8592-0899239d2b69.png">

Reply:
This is because to get timeStamp, the JS Vanilla Date object is created from the day/month/year. Date object is handling some problems with it. For now it's ok.